### ISSUE TITLE
Confusing behaviour of "minimize" button on top of window and doubleclick on tray icon

### ISSUE THREAD

**Issue by user9931** [dedpnd/yaradio-yamusic#16]
Maybe its not a bug but feature.. 
Platform: windows 10, prebuild package from zip archive

With clicking on "minimize" button , apps minimizing to windows taskbar (as usual app).

On double click on tray icon it "closing" to tray (no taskbar icon, if it was minimized it just dissapear from taskbar); so if it was minimized (with icon in taskbar)  it needs to be clicked on tray icon icon double to show a. windows.
And tray icon menu entry behaves same way (like click on tray icon).

**Comment by dedpnd** [comment]
@thehotery This is correct work. I hide it to free up space on the windows taskbar. But I can make it customizable. You need this setup?
